toolkit/docs/formats/imageconfig.md

```diff
@@ -212,6 +212,27 @@ A sample PreInstallScripts entry pointing to two install scripts where one has i
 ],
 ```
 
+### Networks
+
+Kickstart style unattended installations tends to perform network configurations during the installation process to set IP address, configure the hostname, DNS etc. Thus, the Networks entry is added to enable the users to specify the network config parameters. Currently, the Mariner tooling only supports a subset of the kickstart network command options: bootproto, gateway, ip, net mask, DNS and device. Hostname can be configured using the "Hostname" entry of the image config. 
+
+A sample Networks entry pointing to one network configuration:
+``` json
+    "Networks":[
+    {
+        "BootProto": "dhcp",
+        "GateWay":   "192.168.20.4",
+        "Ip":        "192.169.20.148",
+        "NetMask":   "255.255.255.0",
+        "OnBoot":    false,
+        "NameServers": [
+            "192.168.30.23"
+        ],
+        "Device": "eth0"
+    }
+],
+```
+
```

Comment:
Nice entry, thanks for adding that! I'd also enclose keyword/field names in back quotes to make it clear what is a keyword:

```suggestion
### Networks

Kickstart style unattended installations tends to perform network configurations during the installation process to set IP address, configure the hostname, DNS etc. Thus, the `Networks` entry is added to enable the users to specify the network config parameters. Currently, the Mariner tooling only supports a subset of the kickstart network command options: `bootproto`, `gateway`, `ip`, `net mask`, `DNS` and `device`. Hostname can be configured using the `Hostname` entry of the image config. 

A sample Networks entry pointing to one network configuration:
``` json
    "Networks":[
    {
        "BootProto": "dhcp",
        "GateWay":   "192.168.20.4",
        "Ip":        "192.169.20.148",
        "NetMask":   "255.255.255.0",
        "OnBoot":    false,
        "NameServers": [
            "192.168.30.23"
        ],
        "Device": "eth0"
    }
],
```

```